app/ui/src/Components/SettingsMenu/SettingsMenu.tsx

```diff
@@ -35,20 +41,29 @@ function SettingsMenu() {
     );
   }
 
-  function SSHKeyButton() {
-    return (
-      <Button
-        label="SSH key"
-        key="SSH key"
-        onClick={goToUserSSHKeys}
-        Icon={KeyIcon}
-        className={styles.settingButton}
-        align={BUTTON_ALIGN.LEFT}
-      />
-    );
-  }
+  const LogoutButton = () => (
+    <Button
+      label="Sign out"
+      key="Sign out"
+      onClick={() => doLogout()}
```

Comment:
we can avoid this arrow, we can refactor with this:
```suggestion
      onClick={doLogout}
```